cinema/serializers.py

```diff
@@ -1 +1,38 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Genre, Actor, CinemaHall, Movie
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ("id", "name")
+
+
+class ActorSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Actor
+        fields = ("id" ,"first_name", "last_name", "full_name")
```

Comment:
The `full_name` field in `ActorSerializer` is a property, not a model field. Ensure that you define a `SerializerMethodField` or similar to handle this property correctly in the serializer.